sandbox/src/server.ts

```diff
@@ -66,30 +90,63 @@ export async function returnSchema(): Promise<GraphQLSchema> {
 }
 
 async function cohortApiMockserver() {
-  const server = new ApolloServer<MockserverContext>({
-    schema: await returnSchema(),
+    const app = express();
+    // const httpServer = http.createServer(app);
+    const server = new ApolloServer<MockserverContext>({
+        schema: await returnSchema(),
 
-    plugins: [requestLogger, formatResponse],
-  });
+        plugins: [requestLogger, formatResponse/*, ApolloServerPluginDrainHttpServer({httpServer})*/],
+    });
 
   async function start() {
-    const { url } = await startStandaloneServer(server, {
-      context: async () => ({
-        allCohorts: data
-      }),
-      listen: { port },
+    await server.start()
+
+
+    app.get('/_ping', (req,res,next) => {
+        res.json({
+            status: "pass",
+            ping: "pong",
+            service: req.app.locals.app_name,
+            version: req.app.locals.version_info
+        })
+        res.end();
+        next();
     });
-    console.log(`Cohort api mockserver running at ${url}`);
+
+    app.get('/_status', (req, res, next) =>{
+        res.json({
+            status: "pass",
+            ping: "pong",
+            service: req.app.locals.app_name,
+            version: req.app.locals.version_info
+        });
+        res.end();
+        next();
+    });
+
+    app.use(
+        '/',
+        cors<cors.CorsRequest>(),
+        bodyParser.json({ limit: '50mb' }),
```

Comment:
or even 1MB

Reply:
also:
 - 1MB -> one megabyte
 - 1mb -> one milli bit, whatever that is!!!

Reply:
I nabbed that straight from the apollo docs, they obviously have high opinions of their response sizes. A milli-bit is obviously a thousandth of a bit. The question is: How many bits would you need to represent that? And have you just invented a whole new area of Boolean logic? 

In seriousness, I'll change this on Ateeqs branch so that I don't nuke the approvals on this one.